resources/LogicalElement.xml

```diff
@@ -0,0 +1,42 @@
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="LogicalElement"/>
+  <text>
+    <status value="generated"/>
+    <div xmlns="http://www.w3.org/1999/xhtml">
+      <p>LogicalElement to erase id and extension types because they are need in CDA</p>
+    </div>
+  </text>
+  <extension url="http://hl7.org/fhir/StructureDefinition/elementdefinition-namespace">
+    <valueUri value="urn:hl7-org:v3"/>
+  </extension>
+  <url value="http://hl7.org/fhir/cda/StructureDefinition/LogicalElement"/>
+  <name value="LogicalElement"/>
+  <title value="LogicalElement"/>
+  <status value="active"/>
+  <experimental value="false"/>
+  <publisher value="HL7"/>
+  <description value="Defines the basic properties of every data value. This is an abstract type, meaning that no value can be just a data value without belonging to any concrete type. Every concrete type is a specialization of this general abstract DataValue type."/>
+  <kind value="logical"/>
+  <abstract value="true"/>
+  <type value="LogicalElement"/>
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Element"/>
+  <derivation value="constraint"/>
+  <differential>
+    <element id="Element.id">
```

Comment:
Should this be "LogicalElement.id" instead of "Element.id"?

Reply:
No I would like to constrain the base id Element aways, since it conflicts with the V3 id element. This is the most problematic change in this pull request. I found also a zulip discussion, will ask for an update there: https://chat.fhir.org/#narrow/stream/179177-conformance/topic/Logical.20Model.20and.20extension

Reply:
After reviewing the comments from Grahame in the mentioned link above I removed the baseDefinition and constraint from LogicalElement.